book/06-github/sections/1-setting-up-account.asc

```diff
@@ -1,97 +1,97 @@
 
-.The GitHub sign-up form
-image::images/signup.png[The GitHub sign-up form]
+.O formulário de Sign-in do GitHub.
+image::images/signup.png[The GitHub sign-up form.]
 
-The next thing you'll see is the pricing page for upgraded plans, but it's safe to ignore this for now.
-GitHub will send you an email to verify the address you provided.
-Go ahead and do this; it's pretty important (as we'll see later).
+A próxima coisa que você verá é a página de preços para planos melhores, mas pode ignorar isso por enquanto.
+O GitHub vai te enviar um email para verificar o email que você forneceu.
+Vá em frente e verifique, é muito importante (como veremos mais tarde).
 
 [NOTE]
 ====
-GitHub provides almost all of its functionality with free accounts, except some advanced features.
 
-GitHub's paid plans include advanced tools and features as well as increased limits for free services, but we won't be covering those in this book.
-To get more information about available plans and their comparison, visit https://github.com/pricing[].
+O GitHub oferece quase todas as suas funcionalidades com contas gratuitas, exceto algumas funções avançadas.
+Os planos pagos do GitHub incluem ferramentas e funcionalidades avançadas, bem como limites maiores para serviços gratuitos, mas nós não vamos abordá-los neste livro.
 ====
 
-Clicking the Octocat logo at the top-left of the screen will take you to your dashboard page.
-You're now ready to use GitHub.
+Clicando na logo do Octocat no canto superior esquerdo da tela você vai para a página de dashboard.
+Você está pronto para usar o GitHub.
 
-==== SSH Access
+==== Acesso SSH
 
 (((SSH keys, with GitHub)))
-As of right now, you're fully able to connect with Git repositories using the `https://` protocol, authenticating with the username and password you just set up.
-However, to simply clone public projects, you don't even need to sign up - the account we just created comes into play when we fork projects and push to our forks a bit later.
 
-If you'd like to use SSH remotes, you'll need to configure a public key.
-If you don't already have one, see <<ch04-git-server#r_generate_ssh_key>>.
-Open up your account settings using the link at the top-right of the window:
+A partir de agora, você é completamente capaz de se conectar aos repositórios GitHub usando o protocolo  `https://`, autenticando com o nome de usuário e senha que você forneceu.
+Entretanto, para simplesmente clonar repositórios públicos, você não precisa nem mesmo estar logado - a conta que criamos vem à tona quando fazemos fork nos nossos projetos e dermos push em nossos forks mais para frente.
 
-.The ``Account settings'' link
-image::images/account-settings.png[The ``Account settings'' link]
+Se você quiser usar repositórios remotos via SSH, você precisa configurar uma chave pública.
+Se você ainda não tiver uma, veja <<ch04-git-on-the-server#_generate_ssh_key>>.
```

Comment:
Here the anchor must be preserved as in the source. User-defined anchors can not start with a "_".

Reply:
Note: this is likely to be the cause of the failure to compile the book. If the Pull-request is red, this means that the changes introduced some documentation formatting errors that need to be fixed before merging.